tests/esm3/test_esm3_embeddings.py

```
@@ -0,0 +1,235 @@
+        token_count = len(sequence) + 2
+        values = torch.arange(token_count * 3, dtype=torch.float32).reshape(token_count, 3)
+        self.embeddings = values
+        self.hidden_states = torch.stack([values + 100, values + 200])[:, None, :, :] if include_hidden else None
+        sequence_logits = (
+            torch.arange(token_count * 4, dtype=torch.float32).reshape(token_count, 4) if include_logits else None
+        )
+        self.logits = _FakeForwardTrackData(sequence_logits)
+
+
+class _FakeSDKModel:
+    requested_model_names: list[str] = []
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)